### DISCUSSION TITLE
Does Tanstack Start use rspack and rsbuild by default?

### DISCUSSION THREAD

**Discussion by Foxeye-Rinx** [TanStack/router#2582] [Q&A] [answered]
Does Tanstack Start use rspack and rsbuild for file-based routing by default? I didn't find a document about it during the walkthrough of the "Tanstack Start". If not, how to use them in Tanstack Start?

**Comment by SeanCassiere** [comment, marked as answer]
TanStack Start is vite only since Vinxi relies on vite-specific features for its bundling.

**Comment by rohmanhm** [comment]
Hi @SeanCassiere , We're using `rsbuild` for our apps. We loved it so far and we don't have plan to switch. 
Any example of how to use Start with Rsbuild?

**Reply by SeanCassiere** [reply]
@rohmanhm as answered above, TanStack Start is powered by Vinxi which is Vite-only.

Edit: Technically Vite isn't even a direct dependency of TanStack Start, rather it's what Vinxi uses.

Also, TanStack Start is a framework on its own, so the bundler option isn't opened to the user. Unless Vinxi, opens up switching their bundler to something else, I don't see this changing anytime soon.
